sandbox/handlers/config.js

```diff
@@ -4,6 +4,7 @@ const trigger425SendingGroupId = "d895ade5-0029-4fc3-9fb5-86e1e5370854";
 const sendingGroupIdWithMissingTemplates = "c8857ccf-06ec-483f-9b3a-7fc732d9ad48";
 const sendingGroupIdWithDuplicateTemplates = "a3a4e55d-7a21-45a6-9286-8eb595c872a8";
 const sendingGroupIdWithMissingNHSTemplates = "aeb16ab8-cb9c-4d23-92e9-87c78119175c";
+const globalNhsAppSendingGroupId = "00000000-0000-0000-0000-000000000001";
```

Comment:
I don't know if we would have any other global nhsapp templates, but may be good to add free-text into this variable name

Reply:
Done.